spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb

```diff
@@ -3,6 +3,7 @@
   describe "#resource_attributes" do
     let(:expected_attributes) do
       [
+{ name: :id, type: :integer, swagger_type: :integer, example: kind_of(Integer), required: false },
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.